search/keyword_lang_matcher.hpp

```diff
@@ -1,20 +1,23 @@
 #pragma once
+
 #include "search/keyword_matcher.hpp"
 
-#include "std/utility.hpp"
-#include "std/vector.hpp"
+#include <array>
+#include <utility>
+#include <vector>
 
 namespace search
 {
-
 class KeywordLangMatcher
 {
 public:
+  using StringT = KeywordMatcher::StringT;
```

Comment:
Is it possible to get rid of "T" suffixes?

Reply:
done